### PR TITLE
Add support for Broadlink RM4C mini (0x520D)

### PR DESCRIPTION
**Pull request by Sparty117**

### CODE REVIEW

**Comment by unrat** [comment]
Is it just a coincidence that the ID for the RM4 and the RM4**C** is the same, or is this wrong labeled as RM4 mini? As I understand the naming of these Broadlink devices, the 'C' stands for 'no temp data' from the device. Does that somehow can be a problem if the devices with and without temp data have the same ID?

**Comment by Sparty117** [comment]
What we ordered was “BroadLink RM4C Mini IR Black Bean Universal WIFI Remote Controller Receiver” and this is not showing any temperature data in the Broadlink app. My apologies you are correct, the model number should have been RM4C mini. I’ve updated the fork.

 

When we try to add the integration on HA we get “Device not supported” error and the HA logs show the device ID of 0x520D.

File: broadlink/__init__.py [hunk 1]
```diff
@@ -115,6 +115,7 @@
         0x51DA: ("RM4 mini", "Broadlink"),
         0x5209: ("RM4 TV mate", "Broadlink"),
         0x520C: ("RM4 mini", "Broadlink"),
+        0x520D: ("RM4 mini", "Broadlink"),
```
Comment (kdeyev) [inline broadlink/__init__.py]: RM4C mini

**Comment by kdeyev** [comment]
I have faced the same issue and would like to use the fix, once it is merged.

**Comment by felipediel** [comment]
Thank you! Already available in the new_product_ids branch, it comes in the next release. If you want to install the patch before the release:

**Comment by Sparty117** [comment]
Thanks for this. Do you know when the next release is due? I’m on 2022.8.3 and I’m wondering are you referring to the release 2022.9.0?

 

Kind regards

Mark

**Pull request closed without merging**